### PR TITLE
Retry Seed Download

### PR DESCRIPTION
**Pull request by scbedd**
Related to #23326

### CODE REVIEW

File: eng/scripts/seed-virtualenv-wheels.ps1 [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  if ($LASTEXITCODE -eq 0) {
+    break
+  }
+
+  $attempts += 1
+}
+
+if ($LASTEXITCODE -ne 0) {
+  Write-Error "Unable to successfully populate an app data directory."
+  exit 1
```
Comment (benbp) [inline eng/scripts/seed-virtualenv-wheels.ps1:48]:
```suggestion
  exit $LASTEXITCODE
```

**Review by benbp — APPROVED** [review]

**Comment by scbedd** [comment]
This failure is due to an eng/sync-common PR merging. `Analyze` failing is not related to this PR. Overriding and merging.

**Pull request merged**